docs/configuration.md

```diff
@@ -97,6 +97,18 @@ storage:
 		poolname: radospool
 		username: radosuser
 		chunksize: 4194304
+	swift:
+		username: username
+		password: password
+		authurl: https://storage.myprovider.com/v2.0 or https://storage.myprovider.com/v3/auth
```

Comment:
@snayakie We have not  done any test against Openstack keystone v1.0 as currently it is not officially supported by Openstack. It will be much appreciated to do some test and provide us the feedback If you have the related environment on hand.